Libraries/src/Amazon.Lambda.AspNetCoreServer/Internal/SnapStartEmptyLambdaContext.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using Amazon.Lambda.Core;
+
+namespace Amazon.Lambda.AspNetCoreServer.Internal;
+
+internal class SnapStartEmptyLambdaContext : ILambdaContext, ICognitoIdentity, IClientContext
+{
+    private static Dictionary<string, string> _environmentVariables = new();
+
+    // Copied from Amazon.Lambda.RuntimeSupport.LambdaEnvironment to avoid adding
+    // a reference to that project
+    private const string EnvVarFunctionMemorySize = "AWS_LAMBDA_FUNCTION_MEMORY_SIZE";
+    private const string EnvVarFunctionName = "AWS_LAMBDA_FUNCTION_NAME";
+    private const string EnvVarFunctionVersion = "AWS_LAMBDA_FUNCTION_VERSION";
+    private const string EnvVarHandler = "_HANDLER";
```

Comment:
This one is not used and can be removed.

Reply:
removed